cmd/run.go

```diff
@@ -407,13 +385,47 @@ a commandline interface for interacting with it.`,
 			<-sigC
 		}
 
+		reportWg.Wait()
+
 		if engine.IsTainted() {
 			return ExitCode{error: errors.New("some thresholds have failed"), Code: thresholdHaveFailedErrorCode}
 		}
 		return nil
 	},
 }
 
+func reportUsage(execScheduler *local.ExecutionScheduler, wg *sync.WaitGroup) error {
+	execPlan := execScheduler.GetExecutionPlan()
+	executorConfigs := execScheduler.GetExecutorConfigs()
+	execState := execScheduler.GetState()
+
+	executors := make(map[string]int)
+	for _, ec := range executorConfigs {
+		executors[ec.GetType()]++
+	}
+
+	body, err := json.Marshal(map[string]interface{}{
+		"k6_version": consts.FullVersion(),
+		"executors":  executors,
+		"vus_init":   execState.GetInitializedVUsCount(),
+		"vus_max":    lib.GetMaxPossibleVUs(execPlan),
+		"iterations": execState.GetFullIterationCount(),
+		"duration":   execState.GetCurrentTestRunDuration().String(),
+		"goos":       runtime.GOOS,
+		"goarch":     runtime.GOARCH,
+	})
+	if err != nil {
+		return err
+	}
+
+	res, err := http.Post("https://reports.k6.io/", "application/json", bytes.NewBuffer(body))
```

Comment:
This should probably timeout after, say, 1s?

Reply:
`1s` seems to short, but now that this is at the end of the test execution, we can't delay too much. So, say, `3s`?